### DISCUSSION TITLE
adding reactions to the ai generated picture

### DISCUSSION THREAD

**Discussion by LilRaven8** [rocksdanister/lively#2205] [General]
I went to my own wallpaper on my computer and I wanted it to move with the music. it would be cool if you could add it zooming in or out to the music. you could do something like making the ai generated picture move back and forth. You could even do something like the fluids. Honestly I like the one I suggested first, but you could do any of them.

**Comment by rocksdanister** [comment]
I don't think this effect will look good 🤔

A simple bar visualizer in the center will look way better I feel.
